index.html

```diff
@@ -841,9 +841,9 @@
               </li>
               <li>Determine which <a>payment handlers</a> support
               <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <var>data</var> contains filters that exclude the payment
```

Comment:
I think Adrian's concern is the default behavior. Here's a version that endeavors
to capture (what I believe is) the intent, but sticks closer to the language currently
in the spec:

<li>Determine which <a>payment handlers</a> support <var>identifier</var>. 
For each resulting payment handler, if payment method specific capabilities 
supplied by the payment handler do not match those provided by 
<var>data</var>, the payment handler does not match, otherwise the 
payment handler matches.</li>


For each resulting payment handler, if
<var>data</var> contains filters that exclude the payment
handler, then the payment handler does not match, otherwise the
payment handler matches.